code/modules/mob/living/simple_animal/friendly/fox.dm

```diff
@@ -22,12 +22,66 @@
 	var/turns_since_scan = 0
 	var/mob/living/movement_target
 
+/mob/living/simple_animal/pet/fox/New()
+	if(!vore_organs.len)
+		var/datum/belly/B = new /datum/belly(src)
```

Comment:
This is way better than that OOP we used to use. From now on, we're specifying new subtypes by assigning unique variables on creation!